mypy.ini

```diff
@@ -0,0 +1,226 @@
+[mypy]
+
+# For details on each flag, please see the mypy documentation at:
+# https://mypy.readthedocs.io/en/stable/config_file.html#config-file
+
+# Note: The order of flags listed here should match the order used in mypy's
+# documentation to make it easier to find the documentation for each flag.
```

Comment:
I like this. Thanks for adding the reference link. Always takes me 5 min or so to find the reference 😛